Docker_Weaviate.ipynb

```diff
@@ -0,0 +1,236 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Semantic Search using Weaviate and Docker "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial we create a vector store that can be queried using semantic search on a sample dataset composed of transcribed podcasts. The steps will include uploading your data from a local store, and creating a schema as well as an object store.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In your terminal:  \n",
+    "1. Run your virtual environment:  conda activate /Users/your_path/environment_name OR source path_to_your_VR/bin/activate\n",
```

Comment:
I think the language here needs to be improved. 

The canonical conda syntax is `conda activate myenv` where `myenv` can be the name or the path (source: https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#activating-an-environment).

Also,  this line is confusing (`path_to_your_VR`) - what is VR? 

Instructions 2 and 3 are confusing as they look like parts of the same instruction. If they cloned the repo, they would not need to separately download this file. 

I would suggest something like:

```
1. Create and activate a virtual environment, for example using conda or venv
2. Run Weaviate using Docker, for example with `docker-compose up -d`
```